.github/manual-test

```diff
@@ -48,14 +49,102 @@ fi
 
 printf "\nDid everything work as expected? [yN] "
 read -r response
+
+if [ -n "$tmp_dir" ]; then
+  printf "\n***** Deleting test dir %s *****\n" "$tmp_dir"
+  rm -rf "$tmp_dir"
+  printf "\n"
+fi
+
+if [ "$response" != 'y' ]; then
+  echo "Back to fixing"
+  exit 1
+else
+  echo "Yayyyyyyyyyyy"
+fi
+
+printf "\n***** Creating test dirs & resources *****\n"
+tmp_dir=$(mktemp -d -t nativefier-manual-test-auth-XXXXX)
+
+printf "\n***** Building test app *****\n"
+node ./lib/cli.js 'http://httpbin.org/basic-auth/foo/bar' \
+  --basic-auth-username foo \
+  --basic-auth-password bar \
+  --name "app" \
+  "$tmp_dir"
+
+printf "\n***** Test checklist *****
+- Was successfully logged in via HTTP Basic Auth. Should see:
+    {
+      \"authenticated\": true, 
+      \"user\": \"foo\"
+    }
+- Console: no Electron runtime deprecation warnings/error logged
+"
+
+printf "\n***** Running app *****\n"
+if [ "$(uname -s)" = "Darwin" ]; then
+  open -a "$tmp_dir/app-darwin-x64/app.app"
+else
+  "$tmp_dir/app-linux-x64/app"
+fi
+
+printf "\nDid everything work as expected? [yN] "
+read -r response
+
+if [ -n "$tmp_dir" ]; then
+  printf "\n***** Deleting test dir %s *****\n" "$tmp_dir"
+  rm -rf "$tmp_dir"
+  printf "\n"
+fi
+
 if [ "$response" != 'y' ]; then
   echo "Back to fixing"
   exit 1
 else
   echo "Yayyyyyyyyyyy"
 fi
 
+
+printf "\n***** Creating test dirs & resources *****\n"
+tmp_dir=$(mktemp -d -t nativefier-manual-test-auth-prompt-XXXXX)
+
+printf "\n***** Building test app *****\n"
+node ./lib/cli.js 'http://httpbin.org/basic-auth/foo/bar' \
+  --name "app" \
+  "$tmp_dir"
+
+printf "\n***** Test checklist *****
+- Should get a login window to access page.
+    - Username is \"foo\"
+    - Password is \"bar\".
+- Post login, you should see:
+    {
+      \"authenticated\": true, 
+      \"user\": \"foo\"
+    }
+- Console: no Electron runtime deprecation warnings/error logged
+"
+
+printf "\n***** Running app *****\n"
+if [ "$(uname -s)" = "Darwin" ]; then
+  open -a "$tmp_dir/app-darwin-x64/app.app"
+else
+  "$tmp_dir/app-linux-x64/app"
+fi
+
+printf "\nDid everything work as expected? [yN] "
+read -r response
+
```

Comment:
Is this third test really useful? Remember we're in manual land here, we don't want this script to become giant

Reply:
It's as useful as making sure the login dialog works. If that's not a huge worry we can remove.

Reply:
> It's as useful as making sure the login dialog works. If that's not a huge worry we can remove.

Sry, didn't understand it hit the login dialog, I misread it as testing basically nothing. 👍.